README.md

```diff
@@ -65,7 +65,16 @@ Thanks for checking us out! If you're new here, here are some things you should
 ### Join us on slack 💬
 You can sign up [here](https://join.slack.com/t/rubyforgood/shared_invite/zt-21pyz2ab8-H6JgQfGGI0Ab6MfNOZRIQA) and find us in #human-essentials. Many helpful members are available to answer your questions. Just ask, and someone will be there to help you!
 
-##  Getting Started 🛠️
+##  Getting Started (Codespaces) 🛠️
+
```

Comment:
For discussion, from a copyediting pov.   There will be a slight bias to thinking that whatever we put first is the default.  Codespaces are great for the event, of course, but is that what we want throughout the year?? @awwaiid @dorner